### DISCUSSION TITLE
Alternate Flow

### DISCUSSION THREAD

**Discussion by wrexbe** [LostPaul/obsidian-folder-notes#230] [Ideas]
Right now we have folders, and notes. I don't know how far a plugin can go, but imagine this.

Idea:
There is no concept of folders in the UI, just parent notes, and child notes. An item without a sub-item has no expand arrow. An item with a sub-item has an expand arrow.

Implementation:
You can drag anything onto a regular note, to turn it into a folder, with that item inside.
Taking everything out of a folder, converts it back into a note.
The UI no longer has a make folder option.
If you need a folder, you can make a placeholder note, and use it to turn another note into a folder.

This turn it into a folder concept applies to anything, not necessarily a note.

**Comment by LostPaul** [comment]
I'll maybe do this when I've got some time to spare and I've finished the docs.

**Comment by Peeeet** [comment]
Hmmm, I like your thinking (simplifying the end user experience) and it is very "Notion Like", but I feel that it might be tough to implement considering how much of Obsidian - especially the Files sidebar itself - is built around the file / folder hierarchy that Obsidian respects so well.  I'm no expert, but I feel like to code this in a robust enough way (accounting for all edge-cases) would be a LOT of effort for a very subtle looking difference in the end user experience.

Also I kinda like how some folders don't have a note attached to them (like my Daily Notes folder) and this looks different than all of my Folder Notes (most other folders) and just "Plain Notes".

To that end, the Bookmarks sidebar pane of Obsidian could definitely benefit from this kind of thinking.  Not sure if "LostPaul" wants to dive into extending "Folder Note" functionality to that part of Obsidian, but I feel like the Bookmarks sidebar has potential to be the "Logical" view of your vault - complementary to the "Physical" view of the File Explorer sidebar.
